_data/vendors.yml

```diff
@@ -91,6 +91,14 @@
   pricing_source: https://www.dropbox.com/business/pricing
   updated_at: 2018-10-17
 
+- name: Elasticsearch
+  url: https://elastic.co
+  base_pricing: $0
```

Comment:
Would suggest as the free tier isn't generally included that it may be worth basing off their Elastic Cloud offering which is the only thing they give pricing for (https://www.elastic.co/pricing/)